packages/velaux-ui/src/components/WorkflowStudio/type-select.tsx

```diff
@@ -24,44 +24,22 @@ interface DefinitionCatalog {
   definitions: DefinitionBase[];
 }
 
-const defaultDefinitionCatalog: Record<string, string> = {
-  'step-group': 'Group',
-  deploy: 'Delivery',
-  'apply-app': 'Delivery',
-  'apply-deployment': 'Delivery',
-  'apply-component': 'Delivery',
-  'addon-operation': 'Delivery',
-  'deploy-cloud-resource': 'Delivery',
-  'share-cloud-resource': 'Delivery',
-  'export-service': 'Delivery',
-  notification: 'Notification',
-  webhook: 'Notification',
-  'create-config': 'Config Management',
-  'delete-config': 'Config Management',
-  'list-config': 'Config Management',
-  'read-config': 'Config Management',
-  'export-data': 'Config Management',
-  export2config: 'Config Management',
-  export2secret: 'Config Management',
-  suspend: 'Notification',
-};
-
 const defaultCatalog: Record<string, DefinitionCatalog> = {
```

Comment:
Let's change the `catalog` to `category`.

Reply:
sure!
I will replace all the `catalog` with `category`.